### PR TITLE
Update cheminformatics subdomain links

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

**Comment by simonbray** [comment]
I tried to fix the broken links by replacing capital letters like in this PR: https://github.com/usegalaxy-eu/website/pull/483/files

By the way, I don't think this is the root issue. It doesn't explain why for example https://cheminformatics.usegalaxy.eu/root?tool_id=chembl works but https://cheminformatics.usegalaxy.eu/root?tool_id=pubchem doesn't.

Also, when I run the website locally, all the links work fine (also before this PR, so I don't know if this will actually fix anything).

**Comment by simonbray** [comment]
I checked all the tools and it seems only the links to the chembl, packmol and fpocket tools work. Is it the case that the name in the tools.yml has to be the same as the Galaxy tool id?

**Comment by simonbray** [comment]
It was this commit that broke things it seems: https://github.com/usegalaxy-eu/website/commit/7f3bc5148cea7840959d3dbdcda349e1297f1167

**Comment by bgruening** [comment]
uha ... ping @mtekman

**Comment by gmauro** [comment]
I reverted it: e2ee8106207c7dd0490080061774e482f70f9ba5

**Comment by bgruening** [comment]
Did @mtekman had any reasons for this change?

**Comment by mtekman** [comment]
Thanks for the revert @gmauro, @bgruening the change was to fix some broken tool links on the singlecell.usegalaxy.eu side.

For example, by clicking on the "Normalize" button under Scater

Screenshot_2020-09-04 Galaxy Europe

makes the request for: 

> https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/scater_normalize

This tool exists at that link in the toolshed, but for some reason Galaxy cannot launch it as a tool unless I shorten this to:

> https://singlecell.usegalaxy.eu/root?tool_id=scater_normalize






https://toolshed.g2.bx.psu.edu/repos/iuc/scater_normalize/

**Comment by simonbray** [comment]
I think that only links to the repo, not the tool.

This works: https://singlecell.usegalaxy.eu/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/scater_normalize/scater_normalize

**Comment by gmauro** [comment]
https://github.com/usegalaxy-eu/website/blob/master/_data/tools.yml#L206
I guess this is wrong

**Comment by mtekman** [comment]
Yep, let me try to fix this
